BuildScript.Nuke/Build.RunTest.cs

```diff
@@ -24,10 +24,10 @@ public partial class Build : NukeBuild
       .Description("Remove build output, log and temp folders")
       .Executes(() =>
       {
-        var directoryCollection = new DirectoryHelper(RootDirectory);
-        FileSystemTasks.DeleteDirectory(directoryCollection.OutputDirectory);
-        FileSystemTasks.DeleteDirectory(directoryCollection.LogDirectory);
-        FileSystemTasks.DeleteDirectory(directoryCollection.TempDirectory);
+        var directories = new Directories(RootDirectory, BuildProjectDirectory);
+        FileSystemTasks.DeleteDirectory(directories.Output);
+        FileSystemTasks.DeleteDirectory(directories.Log);
+        FileSystemTasks.DeleteDirectory(directories.Temp);
```

Comment:
Can't this be changed to the following?
```suggestion
        FileSystemTasks.DeleteDirectory(Directories.Output);
        FileSystemTasks.DeleteDirectory(Directories.Log);
        FileSystemTasks.DeleteDirectory(Directories.Temp);
```